lessons/Early-Math/computation/langs/en-us/pages/circles-and-computation.adoc

```diff
@@ -0,0 +1,51 @@
+= Circles and Computation
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+
+/* for table cells with immediate .content children, which have immediate
+ * .paragraph children: use flex to space them evenly and center vertically
+*/
+td > .content > .paragraph {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+</style>
+++++
+
+
+@do{
+  (define coe1a '(* (* 30 1/2) (+ 1/2 3/2)))
+  (define coe1b '(* (?ANS 15) (?ANS 2)))
+  (define coe2a '(+ (?ANS 6) (/ 9 0.3)))
+  (define coe2b '(+ 6 (?ANS 30)))
+  (define coe3a '((?ANS /) (+ (?ANS 18) 2) ((?ANS *) 2.5 2)))
+  (define coe3b '(/ 20 (?ANS 5)))
+  (define coe4a '(* ((?ANS +) 4 (- 9 (?ANS 8))) (?ANS 5)))
+  (define coe4b '(* (+ (?ANS 4) 1) 5))
+  (define coe4c '(* (?ANS 5) (?ANS 5)))
+  (define coe5a '(+ (* 6 4) (/ (+ (?ANS 9) 6) 5)))
+  (define coe5b '((?ANS +) (?ANS 24) (/ 15 5)))
+  (define coe5c '(+ (?ANS 24) (?ANS 3)))
+  (define coe6a '((?ANS *) ((?ANS +) (?ANS 1/2) ((?ANS -) (?ANS 3) 1.5)) (+ 5 (* 3 1/3))))
+  (define coe6b '((?ANS *) (+ 1/2 1.5) (+ 5 1)))
+  (define coe6c '(* 2 6))
+}
+
+
+The Circles below represent a step-by-step computation, which results in an answer. Some of the steps are missing numbers! Fill those numbers in so that each sequence of circles will end with the answer shown on the right.
+
+[.FillVerticalSpace, cols="^.^1a,.^14a,stripes="none", options="header"]
+|===
+|    | Circle Solving Diagram
+| @n | @show{(coe coe1a)} &rarr; @show{(coe coe1b)} &rarr; @math{30}
+| @n | @show{(coe coe2a)} &rarr; @show{(coe coe2b)} &rarr; @math{36}
+| @n | @show{(coe coe3a)} &rarr; @show{(coe coe3b)} &rarr; @math{4}
+| @n | @show{(coe coe4a)} &rarr; @show{(coe coe4b)} &rarr; @show{(coe coe4c)} &rarr; @math{25}
+| @n | @show{(coe coe5a)} &rarr; @show{(coe coe5b)} &rarr; @show{(coe coe5c)} &rarr; @math{27}
+| @n | @show{(coe coe6a)} &rarr; @show{(coe coe6b)} &rarr; @show{(coe coe6c)} &rarr; @math{12}
+|===
+
+
```

Comment:
In some math curricula (cmp for example) circles of evaluation might be useful well before students learn to multiply fractions or decimals. And since working with fractions and decimals are also often introduced in two separate units, you might even want to disaggregate them.

I would suggest making multiple versions of this workbook page: 1 with whole numbers, others with fractions/decimals/integers.